app/bundles/EmailBundle/Tests/Model/SendEmailToContactTest.php

```diff
@@ -725,4 +755,98 @@ function () use ($emailMock) {
         $errorMessages = $model->getErrors();
         $this->assertCount(1, $errorMessages);
     }
+
+    public function testSetSampleMailerByTemporaryMailerSpoolFile()
+    {
+        $this->testSetSampleMailerByTemporaryMailerSpoolMemory('file');
```

Comment:
This is interesting, but I think it cannot work:
1) You need different behaviour for each type, right? So you need to write a little different test for each type.
2) Even if the behaviour is same, you can use data providers: https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers